README.md

```diff
@@ -17,7 +17,7 @@ in the shell you are applying this from.
 ## Requirements not managed here
 
 - A kubernetes cluster
-- An ingress controller in said cluster
+- An nginx ingress controller in said cluster
```

Comment:
TIL an ingress controller is different from an ingress..  but I still can't figure out how to install such a controller, neither from https://github.com/nginxinc/kubernetes-ingress/tree/master/nginx-controller nor https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-controllers.